bindata/network/multus/002-rbac.yaml

```diff
@@ -53,26 +54,95 @@ rules:
   - create
   - patch
   - update
+- apiGroups: ["certificates.k8s.io"]
+  resources:
+    - certificatesigningrequests
+  verbs:
+    - create
+    - get
+    - list
+    - watch
+
+---
+# This role is ancillary tools in the multus namespace.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: multus-ancillary-tools
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["k8s.cni.cncf.io"]
+  resources: ["*"]
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - nodes
+  verbs:
+  - get
+- apiGroups: ["", "events.k8s.io"]
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multus-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: multus-proper
```

Comment:
This seems to be enabled unconditionally but we should allow for the fallback option that would disable the per-node cert/webhook.
See here: https://github.com/openshift/cluster-network-operator/pull/1983/files#diff-cd68b4caec2e8a6afb4c120d980469834c62524b90a300f2adf90b6d6bec0c55R49-R58
And: https://github.com/openshift/cluster-network-operator/pull/1983/files#diff-13e60e30d72f7b5c263fabc48ae9533c32531df3983d7e70d8a4e1a0a1c1a97dR25-R28

Reply:
Good call. Added conditions for RBAC + serviceaccount + serviceaccountname, as well as multus configuration to disable cert per node mode.